radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/P2PConfig.java

```diff
@@ -308,6 +315,14 @@ private NodeId parseNodeId(String s) {
           "peerAllowList is non empty but will be ignored because usePeerAllowList is not set");
     }
 
+    if (config.useProxies() && config.authorizedProxies().isEmpty()) {
+      throw new IllegalArgumentException("authorizedProxies can't be empty if useProxies is true");
+    }
+
+    if (!config.useProxies() && !config.authorizedProxies().isEmpty()) {
+      throw new IllegalArgumentException("authorizedProxies can't be set if useProxies is false");
```

Comment:
While I understand the intent of this check, I think it might be inconvenient in practice. 
For example, if a user had configured proxies and then decided to just disable them (for example, experimenting or checking something), he will ought to clear the configuration. But presence of the proxies if they are disabled should not prevent the node from start.

Reply:
yeah, I think you're right. I'll change that to just log a warning